identity/app/services/MembersDataApiService.scala

```diff
@@ -0,0 +1,80 @@
+package services
+
+import play.api.libs.ws.{DefaultWSCookie, WSClient, WSCookie, WSResponse}
+import play.api.mvc.{Cookie, Cookies}
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import utils.SafeLogging
+import scala.concurrent.{ExecutionContext, Future}
+
+// Modeled on members-data-api Attributes - ContentAccess
+// https://github.com/guardian/members-data-api/blob/master/membership-attribute-service/app/models/Attributes.scala
+case class ContentAccess(
+  isMember: Boolean,
```

Comment:
No need for `is/has` prefixes. Then we can provide codecs with `Json.format[ContentAccess]`